internal/provider/kubernetes/controller_test.go

```diff
@@ -318,7 +317,7 @@ func TestEnqueueManagedClass(t *testing.T) {
 		tc := testCases[i]
 
 		// Create the reconciler.
-		logger := logging.DefaultLogger(v1alpha1.LogLevelInfo)
+		logger := logging.DefaultLogger(egcfgv1a1.LogLevelInfo)
```

Comment:
Is this a manual change, or does this linter help ensure that named imports share the same name across files?

Reply:
no, just ensure you cannot import same package more than once.